src/lang/en.json

```diff
@@ -59,7 +59,7 @@
         "BEN": "BC Benefit Company",
         "BC": "BC Limited Company",
         "CP": "BC Cooperative Association",
-        "SP": "Sole Proprietorship",
+        "SP": "BC Sole Proprietorship",
```

Comment:
@kialj876 do you know what is proper translation for this, is it with BC, as is now in prod, or without ?
or should I bring it up with business ppl ?

Reply:
I think with 'BC' is correct